### PR TITLE
ZEPPELIN213: Notebook toggleAllTable and toggleAllEditor does not mat…

### PR DESCRIPTION
**Pull request by eranwitkon**
This fix match the notebook buttons (code & output) behavior to the paragraph behavior (code & output).
BEFORE:
selection_008

AFTER:
selection_009

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Tested and LGTM

File: zeppelin-web/src/index.html [hunk 1]
```diff
@@ -102,7 +102,7 @@
     <script src="bower_components/angular-xeditable/dist/js/xeditable.js"></script>
     <script src="bower_components/highlightjs/highlight.pack.js"></script>
     <script src="bower_components/lodash/lodash.js"></script>
-    <script src="bower_components/angular-filter/dist/angular-filter.js"></script>
+    <script src="bower_components/angular-filter/dist/angular-filter.min.js"></script>
```
Comment (felixcheung) [inline zeppelin-web/src/index.html]: any idea why this keeps switching? I see this locally too

**Comment by felixcheung** [comment]
Very cool. LGTM.

**Comment by tzolov** [comment]
+1

**Pull request closed without merging**